web/views.py

```
@@ -125,6 +125,7 @@
     TeamInvite,
     TimeSlot,
     WebRequest,
+    SocialShareDiscount,
```

Ruff (0.8.2)

`.models.SocialShareDiscount` imported but unused

Remove unused import: `.models.SocialShareDiscount`

(F401)